tic_tac_toe/src/board.py

```diff
@@ -0,0 +1,90 @@
+# Board Representation
+
+# test_board = [0, 1, -1, 0, 1, -1, 0, 1, -1]
+# 9 values, for 9 spaces on the board
+#
+# -1 = X
+# 0 = No Value
+# 1 = O
+#
+
+
+class Board:
+    def __init__(self, dim):
+        self.values = [None, None, None, None, None, None, None, None, None]
+
+        self.template = "{}|{}|{}\n-------\n{}|{}|{}\n-------\n{}|{}|{}"
+
+        self.dim = dim
+
+        self.winner = None
+
+    def _convert(self, x, y):
+        return (x*int(self.dim))+y
+
+    def render_board(self):
+        return self.template.format(*self.values)
+
+    def raw_board(self):
+        return self.values
+
+    def update_values(self, x, y, value):
+
+        index = self._convert(x, y)
+
+        self.values[int(index)] = value
+
+    def get_value(self, x, y):
+        
+        1d = self._convert(x, y)
+
+        return self.values[1d]
+
+    def get_move(self, player):
```

Comment:
This shouldn't be in this class.  You should keep user interactions with the board separate from the data and functions which operate on the board.